guides/source/configuring.md

```diff
@@ -952,6 +952,16 @@ text/javascript image/svg+xml application/postscript application/x-shockwave-fla
 
 * `config.active_storage.draw_routes` can be used to toggle Active Storage route generation. The default is `true`.
 
+* `config.active_storage.resolve_name` can be used to globally change how Active Storage files are delivered.
```

Comment:
I like this idea, but we can be a bit more explicit: `config.active_storage.resolve_model_to_route`.